webapp/models.py

```diff
@@ -38,7 +38,7 @@ def as_json(self):
             **self.data,
             "created": self.created.strftime("%a, %d %b %Y %H:%M:%S"),
             "file_path": self.file_path,
-            "tags": ", ".join(self.tags),
+            "tags": [tag.name for tag in self.tags],
```

Comment:
This is an API change isn't it? You're changing a String type into a List type? Or did I misunderstand?

If I'm right, might that not break clients like the assets manager?

Reply:
Well seen! the original code was failing so I was trying to fix the error but missed the format, this should be:
```python
            "tags": ", ".join([tag.name for tag in self.tags]),
```

Reply:
@goulinkh are there more updates to be made or can this be merged?

Reply:
@goulinkh bump - do we need to do anything with this?

Reply:
Sorry @sowasred2012 I was off last week, I will update this PR as soon as possible.

Reply:
Trying to connect to the production Swift server to properly QA this PR...